app/src/main/java/com/jerboa/ui/components/inbox/InboxActivity.kt

```diff
@@ -143,7 +144,7 @@ enum class InboxTab {
     Messages,
 }
 
-@OptIn(ExperimentalPagerApi::class)
+@OptIn(ExperimentalFoundationApi::class, ExperimentalMaterialApi::class)
```

Comment:
Double :) . Android studio does this to me all the time too.

Reply:
These are separate, removing either causes a build failure. Is there an opt-in that is a superset of these two?

Reply:
Ohhh my bad, I thought they were the same string at first glance.